eng/pipelines/templates/stages/archetype-java-release.yml

```
@@ -116,7 +116,38 @@ stages:
                     PackageName: ${{artifact.name}}
                     ServiceName: ${{parameters.ServiceDirectory}}
                     ForRelease: true
-
+          - job: InstallationCheck
+            displayName: "Installation Check"
+            variables:
+              - template: ../variables/globals.yml
+            steps:
+              - download: current
+                artifact: ${{parameters.ArtifactName}}-signed
+              - pwsh: |
+                  Copy-Item "$(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed/${{artifact.groupId}}/${{artifact.name}}/" "$(Pipeline.Workspace)/install-check-dir/${{artifact.name}}" -Recurse
+                  Write-Host "Copying artifact ${{artifact.name}}"
+                  $pomf = Get-ChildItem $(Pipeline.Workspace)/install-check-dir/${{artifact.name}}/*.pom
+                  Rename-Item "$($pomf)" pom.xml
```

If you don't stringify it and pass the file object to `Rename-Item`, it will handle failure cases more predictably (for example if you find multiple files, the failed command will be too many parameters, rather than a filename with spaces that doesn't exist).

```suggestion
                  Rename-Item $pomf pom.xml
```